app/views/patients/_pledge.html.erb

```diff
@@ -1,6 +1,8 @@
-<%= render '/shared/multi_modal', disable_next: disable_continue?(patient), :captured => capture { %>
+
 
 <% if not patient.pledge_sent %>
+
+  <%= render '/shared/multi_modal', disable_next: disable_continue?(patient), :captured => capture { %>
```

Comment:
Looks like you swapped this around to just the multimodal on the submit pledge case. Not quite sure I follow why you did that, can you give me a sense of how this played out?

Reply:
Since this is a one step modal process the multi_modal partial doesn't work very well with it. 

The multi_modal footer has two buttons, a cancel and a next-step/complete button. Based on Molly's design it seemed like we wanted a footer with just a 'No' and 'Yes' button for cancelling the pledge, with the 'No' button closing the modal and the 'Yes' button submitting the :pledge_sent value of false. 

The multi_modal design has the form in the body of the modal and the footer has buttons for going back and forth through the modal steps and for closing the modal. The cancel pledge modal is only going to be one step, so we don't need the multi_modal footers to get to other steps in the process.